app/src/gecko/java/org/mozilla/focus/web/Config.kt

```diff
@@ -0,0 +1,5 @@
+package org.mozilla.focus.web
```

Comment:
nit: license header here and in the other new files

```Kotlin
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
I'll add the license header.